app/views/admin/splits/index.html.erb

```diff
@@ -1,19 +1,36 @@
-<div class="TakeoverText">
-  <h1 class="TakeoverText-title sc-m-v--l">Active Split Tests</h1>
+<div class="TakeoverText sc-m-v--l">
+  <h1 class="TakeoverText-title">
+    Active Split Tests
+    <% if @app %>
+      for <%= @app.name %>
+    <% end %>
+  </h1>
+  <% if App.by_name.any? %>
+    <select onchange="if (this.value) window.location.href=this.value" data-testId="appSelector">
+      <option value="">Filter by App</option>
+      <% App.by_name.each do |app| %>
+        <option value="<%= admin_root_path(app: app.name) %>"><%= app.name %></option>
+      <% end %>
+    </select>
```

Comment:
Can this use a regular form GET submission instead of this inline change handler? Redirecting based on selecting an item has unexpected accessibility issues. With a form submission, we don't need to provide the whole path as the value for each option either, just set the name and id on the select to `app` and the value of each option to `app.name`. 

The select should have an associated label as well rather than using an empty option as the identifier.

Reply:
<img width="415" alt="Screen Shot 2023-02-28 at 3 11 23 PM" src="https://user-images.githubusercontent.com/606/222003695-4fc34674-9953-4884-a8e4-c7815d67eb7e.png">